lib/action_api/request.ex

```diff
@@ -47,7 +47,12 @@ defmodule GraphConn.ActionApi.Request do
           params: map()
         }) :: String.t()
   defp _calculate_id(payload) do
-    "#{payload.ticket_id}#{payload.handler}#{payload.capability}#{inspect(payload.params)}"
+    context_id =
+      "#{payload.handler}#{payload.capability}#{inspect(payload.params)}"
```

Comment:
you removed {payload.ticket_id} is it not required to make it unique?

Reply:
it's still in there but added without hashing - should still be small enough but this way hash collisions would have to occur inside of same issue... so I though it would be safer to exclude it from the hashing altogether